argo-workflow/argo-install.yaml

```diff
@@ -240,9 +244,8 @@ spec:
       - args:
         - server
         - --namespaced
-#        - --auth-mode 
-#        - client
-        image: argoproj/argocli:v2.11.7
+        - --secure=false
```

Comment:
TLS is enabled by default since 3.0.0
https://github.com/argoproj/argo-workflows/releases/tag/v3.0.1